config/routes.rb

```diff
@@ -74,6 +74,11 @@
   #post "object/:id/upload", :to => 'file#create', :as => 'upload'
   post "object/:id/deriv/:ds", :to => 'file#deriv', :as => 'deriv'
   get "object/:id/zoom/:cmp", :to => 'dams_objects#zoom', :as => 'zoom'
+
+  # EMBED UI
+  get "embed/:id/:cmp", :to => 'dams_objects#embed', :as => 'embed'
```

Comment:
Maybe we should follow the same pattern as `zoom` to use `object/:id/embed/:cmp` since the `:id` is object id? Just a though.

Reply:
Since this is the URL we are actually sharing via the Embed UI, I want it to be short, logical, and identifiable. That is, I don't want there to be any confusion on the user's part and I want it to display well in the UI, so I specifically designed it to read as it does (I.e., "embed/:id/:cmp").

The pattern is short, it presents the action via the term "embed" immediately, and it says what it is embedding via the IDs following it.

E.g., "https://library.ucsd.edu/dc/embed/uv8022051s/0"

It also follows a similar design pattern used elsewhere across the web, so users will likely already be familiar with it.

E.g., <website>/embed/<id>

Thank you for the suggestion, but let's keep this one as is.

Reply:
That makes senses and I am fine with it. Thanks.